include/pmacc/types.hpp

```diff
@@ -23,6 +23,39 @@
 
 #pragma once
 
+#include <cupla/types.hpp>
+
+#ifndef PMACC_CUDA_ENABLED
+#   define PMACC_CUDA_ENABLED ALPAKA_ACC_GPU_CUDA_ENABLED
+#endif
+
+#if( PMACC_CUDA_ENABLED == 1 )
+/* include mallocMC before cupla renaming is activated, else we need the variable acc
+ * to call atomic cuda functions
+ */
```

Comment:
Using mallocMC only for cuda and the dependence of mallocMC for cuda (?) will get us in trouble if the Alpaka HIP back end is ready to be used as we also want to have mallocMC for HIP.